ci/ci.sh

```diff
@@ -3,6 +3,8 @@ export RUN_ID=$(node generate-id.js)
 echo "INFO: Run id: $RUN_ID"
 echo "INFO: Sleep seconds set for services to start: ${SLEEP:=60}"  # defaults to 60 seconds
 
+export S3_DIRECTORY_NAME=$(date +%s)
```

Comment:
@v-stickykeys, should we use millisecond precision so we can avoid practically all conflicts in case multiple tests are initiated in parallel, whether through local testing or in infra by any user?

Reply:
Though I guess that may not be reliably possible through bash...